python/dgl/distributed/dist_graph.py

```diff
@@ -336,6 +336,14 @@ def __init__(self, server_id, ip_config, num_servers,
             self.client_g, _, _, self.gpb, graph_name, \
                     ntypes, etypes = load_partition(part_config, self.part_id, load_feats=False)
             print('load ' + graph_name)
+            # formatting dtype
```

Comment:
One more thing. Could you add a TODO here that this is a temporary fix and what should be the better solution? Start with `TODO(rei):` so ppl will contact you if they have questions.